hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -61,6 +61,23 @@ public HdfsFrameworkConfig() {
     setConf(configuration);
   }
 
+  public String getPrincipal() {
+    return getConf().get("mesos.hdfs.principal", "");
+  }
+
+  public String getSecret() {
+    return getConf().get("mesos.hdfs.secret", "");
+  }
+
+  public boolean cramCredentialsEnabled() {
+    String principal = getPrincipal();
+    String secret = getSecret();
```

Comment:
I got rid of the null check altogether.  Not needed since the Configuration get method provides a guaranteed default value if the config entry isn't present.